openfisca_us/variables/gov/states/md/tax/income/additions/md_lump_sum_retirement_distribution.py

```diff
@@ -0,0 +1,18 @@
+from openfisca_us.model_api import *
+
+
+class md_lump_sum_retirement_distribution(Variable):
```

Comment:
remove these additions from this pr

Reply:
Yikes. I really did some damage with my merges